packages/a11y/src/test/clear.test.js

```diff
@@ -16,7 +16,7 @@ describe( 'clear', () => {
 		document.body.appendChild( container2 );
 
 		clear();
-		expect( container1.textContent ).toBe( '' );
-		expect( container2.textContent ).toBe( '' );
+		expect( container1 ).toHaveTextContent( '' );
+		expect( container2 ).toHaveTextContent( '' );
```

Comment:
Good morning @tyxla :wave: using the assertion with an empty string is not good. As the assertion checks for substrings by default, it will always match! Fortunately the matcher is smart enough to detect this and when a non-empty string is being matched against `''`:
```js
expect( elWithActualText ).toHaveTextContent( '' );
```
it will fail with some advice:
```
Checking with empty string will always match, use .toBeEmptyDOMElement() instead:
```

Reply:
That makes sense @jsnajdr, thanks for pointing it out. Will follow-up (if someone hasn't addressed it yet).